### PR TITLE
Add index periodical job for cert-manager/website

### PR DESCRIPTION
**Pull request by meyskens**
This adds a periodical job to update the search index of cert-manager.io

### CODE REVIEW

File: config/jobs/cert-manager/website/cert-manager-website-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - scripts/index
+      env:
+      - name: ALGOLIA_WRITE_KEY
+        valueFrom:
+          secretKeyRef:
+            name: cert-mamager-website-algolia-key
+            key: ALGOLIA_WRITE_KEY
+      resources:
+        requests:
+          cpu: 2
```
Comment (munnerz) [inline config/jobs/cert-manager/website/cert-manager-website-periodics.yaml]: The VMs in this build cluster only have 2 CPUs, so probably best to set this to `cpu: 1` instead 😄 - I think you should also be able to reduce the memory request a bit? Perhaps `1Gi` or `2Gi`?

File: config/jobs/cert-manager/website/cert-manager-website-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    containers:
+    - image: "eu.gcr.io/jetstack-build-infra-images/golang-nodejs:20191129-c49853e-1.13.4"
+      args:
+      - bash
+      - scripts/index
+      env:
+      - name: ALGOLIA_WRITE_KEY
+        valueFrom:
+          secretKeyRef:
+            name: cert-mamager-website-algolia-key
```
Comment (munnerz) [inline config/jobs/cert-manager/website/cert-manager-website-periodics.yaml]: manager not mamager 😄

Comment (meyskens) [reply config/jobs/cert-manager/website/cert-manager-website-periodics.yaml]: oops!

File: config/jobs/cert-manager/website/cert-manager-website-periodics.yaml (hunk 1 above)
Comment (meyskens) [reply config/jobs/cert-manager/website/cert-manager-website-periodics.yaml]: Done, would stay at 2 since we run node.js

**Pull request merged**